test/e2e/BUILD

```diff
@@ -46,10 +46,8 @@ go_library(
     name = "go_default_library",
     srcs = [
         "addon_update.go",
-        "apparmor.go",
         "audit.go",
```

Comment:
either is fine with me. Major part of the implementation is apimachinery, but on a higher level sig-auth fits better.